devel/lazarus/Portfile

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                lazarus
+version             2.0.4
+revision            1
+categories          devel
+platforms           darwin
+license             GPL-2 LGPL-2
+maintainers         {@kamischi web.de:karl-michael.schindler} openmaintainer
+description         Free Pascal IDE.
+long_description    Lazarus is an open-source development system that builds \
+                    on the Free Pascal compiler by adding an integrated \
+                    development environment (IDE).  It includes a \
+                    syntax-highlighting code editor and a visual form \
+                    designer, as well as a component library that is highly \
+                    compatible with Delphi's Visual Component Library \
+                    (VCL).  The Lazarus Component Library (LCL) includes \
+                    equivalents for many of the familiar VCL controls such \
+                    as forms, buttons, text boxes and so on, which are used \
+                    to create applications with a graphical user interface \
+                    (GUI).
+
+homepage            https://wiki.freepascal.org/Main_Page
+master_sites        sourceforge:lazarus
+set src             ${name}-${version}.tar.gz
```

Comment:
This isn't the default for sourceforge already?

Reply:
I'll check. Could be.

Reply:
I'm not understanding the purpose of this variable? You are setting the new variable `src` to the default value for the existing global variable `distfiles`.

Reply:
Probably a leftover from another example. I removed it and it still works. Thanks for the hint.